mjpc/task.h

```diff
@@ -60,7 +60,7 @@ class Task {
   void Residuals(const mjModel* m, const mjData* d, double* residuals) const;
 
   // apply transition function
-  void Transition(const mjModel* m, mjData* d);
+  void Transition(const mjModel* m, mjData* d) const;
```

Comment:
Is this the right thing to do here? This was originally added by @thowell so I'm not sure.